examples/context/multiple-contexts.js

```diff
@@ -1,7 +1,7 @@
-// Theme context, default to light theme
-const ThemeContext = React.createContext('light');
+// Contexte thème, par défaut à clair 
+const ThemeContext = React.createContext('clair');
 
-// Signed-in user context
+// Contexte utilisateur authentifié
 const UserContext = React.createContext({
   name: 'Guest',
```

Comment:
```suggestion
  name: 'Invité',
```

Là pour le coup c'est clairement un texte affiché, on traduit…